<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userinfocard</class>
 <widget class="QWidget" name="userinfocard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>lion</string>
   </property>
  </widget>
  <widget class="QLabel" name="sex_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>male</string>
   </property>
  </widget>
  <widget class="QLabel" name="profession_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>student</string>
   </property>
  </widget>
  <widget class="QLabel" name="location_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>广东珠海</string>
   </property>
  </widget>
  <widget class="QLabel" name="birth_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>13-5-2015</string>
   </property>
  </widget>
  <widget class="QLabel" name="hobby_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
